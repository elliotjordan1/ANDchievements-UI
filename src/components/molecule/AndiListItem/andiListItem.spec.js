import React from 'react';
import { render } from '@testing-library/react';
import Component from '.';

describe('AndiListItem component', () => {
  it('renders correctly', () => {
    const andi = {
      imageURL : 'http://image.url',
      name: 'Jose',
      andiProjectRole: 'PD'
    };

    const component = <Component andi={andi} />;

    const wrapper = render(component);
    
    expect(wrapper).toMatchSnapshot();
  });
});
