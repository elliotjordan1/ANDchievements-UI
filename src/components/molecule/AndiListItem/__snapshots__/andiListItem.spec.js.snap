// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AndiListItem component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="VerticalListItem-sc-1q5r9eq-0 esOjNi"
      >
        <div
          class="Icon-e1hn6e-0 hxRnsx"
        />
        <div
          class="ListDescription-sc-1tshlis-0 linVKL"
        >
          Jose
          <em>
            PD
          </em>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="VerticalListItem-sc-1q5r9eq-0 esOjNi"
    >
      <div
        class="Icon-e1hn6e-0 hxRnsx"
      />
      <div
        class="ListDescription-sc-1tshlis-0 linVKL"
      >
        Jose
        <em>
          PD
        </em>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
